src/components/LibraryItem.tsx

```diff
@@ -370,12 +392,22 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     onLibraryItemClicked() {
         // Toggle expansion state.
         let currentlyExpanded = this.state.expanded;
+
         if (this.props.data.childItems.length > 0 && !currentlyExpanded && this.props.onItemWillExpand) {
             this.props.onItemWillExpand(ReactDOM.findDOMNode(this));
         }
 
         this.setState({ expanded: !currentlyExpanded });
 
+        //auto expand the coregroup elements
+        if(this.props.data.itemType === "category" ) {
```

Comment:
does this work for nested categories?

Reply:
@Racel curious if this case was considered?

Reply:
@mjkkirschner Only needed for top level categories.